ci/ci/ci.py

```diff
@@ -185,6 +186,34 @@ async def get_pr(request: web.Request, userdata: UserData) -> web.Response:
             artifacts_uri = f'{STORAGE_URI}/build/{batch.attributes["token"]}'
             page_context['artifacts_uri'] = artifacts_uri
             page_context['artifacts_url'] = storage_uri_to_url(artifacts_uri)
+
+            if CLOUD == 'gcp':
+                start_time = status['time_created']
+                end_time = status['time_completed']
+                assert start_time is not None
+                assert PROJECT is not None
+
+                page_context['logging_queries'] = {
+                    'batch-k8s-error-warning': generate_gcp_service_logging_url(
+                        PROJECT,
+                        ['batch', 'batch-driver'],
+                        DEFAULT_NAMESPACE,
+                        start_time,
+                        end_time,
+                        ['ERROR', 'WARNING'],
+                    ),
+                    'batch-workers-error-warning': generate_gcp_worker_logging_url(
+                        PROJECT, DEFAULT_NAMESPACE, start_time, end_time, ['ERROR', 'WARNING']
+                    ),
+                    'ci-k8s-error-warning': generate_gcp_service_logging_url(
+                        PROJECT, ['ci'], DEFAULT_NAMESPACE, start_time, end_time, ['ERROR', 'WARNING']
+                    ),
+                    'auth-k8s-error-warning': generate_gcp_service_logging_url(
+                        PROJECT, ['auth', 'auth-driver'], DEFAULT_NAMESPACE, start_time, end_time, ['ERROR', 'WARNING']
+                    ),
```

Comment:
I find this a bit noisy for this function, can we package these lines in a function that's also in the utils file and then change this to just be `page_context['logging_queries'] = gcp_log_urls(DEFAULT_NAMESPACE)` (or something to that effect)?